skrf/circuit.py

```diff
@@ -1290,3 +1299,115 @@ def voltages_external(self, power: NumberLike, phase: NumberLike) -> np.ndarray:
         for port_idx in self.port_indexes:
             Vs.append((a[port_idx] + b[:,port_idx])*np.sqrt(z0s[:,port_idx]))
         return np.array(Vs).T
+
+    def plot_graph(self, **kwargs):
```

Comment:
Autocompletion and typing support works only for the `real` methods and not for the dynamically added methods, so this would be the prefered way. iPython autosuggestion works in both cases.